jest.config.js

```diff
@@ -1,9 +1,131 @@
 module.exports = {
-  testMatch: ['**/+(*.)+(spec|test).+(ts|js)?(x)'],
-  transform: {
-    '^.+\\.(ts|js|html)$': 'ts-jest',
-  },
-  resolver: '@nrwl/jest/plugins/resolver',
-  moduleFileExtensions: ['ts', 'js', 'html'],
-  coverageReporters: ['html'],
+  projects: [
+    '<rootDir>/apps/air-discount-scheme/api',
```

Comment:
Is this autogenerated by nx?

If not, would be nicer to create an alias resolver that resolves path aliases from the root tsconfig.

Reply:
It is autogenerated by NX.

Unfortunately, it always appends instead of sorting the array, so we'll see merge conflicts again when two PRs add a lib. Until someone (😅) writes another prettier plugin, like we did for workspace.json and tsconfig.